owncloudApp/src/main/res/values/strings.xml

```diff
@@ -76,6 +76,7 @@
     <string name="prefs_show_hidden_files">Show hidden files</string>
     <string name="prefs_log_open_logs_list_view">Log files</string>
     <string name="prefs_log_no_logs_list_view">Empty log folder!</string>
+    <string name="prefs_log_empty_subtitle">If you enable logging, log files will show up here</string>
```

Comment:
```suggestion
    <string name="prefs_log_empty_subtitle">Enable logging and log files will show up here</string>
```
I think this way it is more consistent with other strings such as `upload_list_empty_subtitle` 👍